MaterialComponents.podspec

```diff
@@ -269,6 +269,15 @@ Pod::Spec.new do |s|
     ss.dependency "MaterialComponents/private/RTL"
   end
 
+  s.subspec "Themes" do |ss|
+    ss.ios.deployment_target = '8.0'
+    ss.public_header_files = "components/#{ss.base_name}/src/*.h"
+    ss.source_files = "components/#{ss.base_name}/src/*.{h,m}", "components/#{ss.base_name}/src/private/*.{h,m}"
+    ss.resources = ["components/#{ss.base_name}/src/Material#{ss.base_name}.bundle"]
+
+    ss.dependency "MaterialComponents/FeatureHighlight"
```

Comment:
This makes the themes dependent on FeatureHighlight. This is not so scalable if the themes depend on all the components.

The best alternative I can come up with is a separate subspec for the themer.